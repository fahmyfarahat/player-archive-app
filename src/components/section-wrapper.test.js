import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import SectionWrapper from "./section-wrapper";

describe("SectionWrapper - general", () => {
  let wrapper;
  beforeEach(()=> {
    const props = {
      className: "test"
    }
    wrapper = shallow(<SectionWrapper {...props}><div>test</div></SectionWrapper>);
  });

  test("without crashing", () => {
    expect(wrapper).toHaveLength(1);
  });

  test("should match Snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
