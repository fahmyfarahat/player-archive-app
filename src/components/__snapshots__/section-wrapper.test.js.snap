// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionWrapper - general should match Snapshot 1`] = `
<section
  className="test"
>
  <Container
    fluid={false}
  >
    <div>
      test
    </div>
  </Container>
</section>
`;
